### PR TITLE
Build umis for py3

### PR DESCRIPTION
**Pull request by jenzopr**
* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [X] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

Currently, umis is restricted to py27. This renders a usage within snakemake impossible. Since I was able to install umis==0.7.0 in py3 via pip, building for py3 in general should not be a problem.

### CODE REVIEW

**Comment by jenzopr** [comment]
Problem is described here http://click.pocoo.org/5/python3/#python3-surrogates , as well as solutions.

**Comment by daler** [comment]
@jenzopr maybe try `en_US.utf8`? 

Also it's likely you'll also need the extended container for the docker image to work. See the [`multiqc` recipe](https://github.com/bioconda/bioconda-recipes/blob/master/recipes/multiqc/meta.yaml#L132) for how to set this up, and [bioconda troubleshooting docs](https://bioconda.github.io/troubleshooting.html#using-the-extended-image) for background.

**Comment by jenzopr** [comment]
Thank you @daler ! I think your hints brought me onto the right track 😃

File: recipes/umis/meta.yaml [hunk 1]
```diff
@@ -37,9 +36,15 @@ requirements:
 test:
   commands:
     # click requires a unicode locale
-    - LANG=C.UTF-8 umis --help
+    - LANG=en_US.UTF-8 umis --help
```
Comment (jenzopr) [inline recipes/umis/meta.yaml]: Maybe exporting `LANG=en_US.UTF-8` is not even needed for the extended container.

Comment (bgruening) [reply recipes/umis/meta.yaml]: Yes, I don't think it is needed.

**Comment by jenzopr** [comment]
Thanks, Björn. If no further comments arise, I'll merge 😃

**Comment by bgruening** [comment]
Go for it - and thanks a lot!

**Pull request merged**